src/ethereum_test_tools/tests/test_code.py

```diff
@@ -96,7 +98,8 @@ def test_yul():
         expected_bytecode += bytes.fromhex("80")
         expected_bytecode += bytes.fromhex("55")
 
-    assert Yul(long_code).assemble() == expected_bytecode
+    # TODO(dan): workaround until it's understood why Shanghai takes so long to compile
+    assert Yul(long_code, fork=Merge).assemble() == expected_bytecode
```

Comment:
Maybe we found an issue in the solc ?

Reply:
Thanks for bringing this up. Could be, need to look into this! Made a ticket so not to forget #148.